### DISCUSSION TITLE
enabling vue.js devtools on quasar 1.1.3

### DISCUSSION THREAD

**Discussion by prescriptionlifeline** [quasarframework/quasar#13034] [General - Components / Directives / etc]
I'd like to use [Vue.js devtools](https://chrome.google.com/webstore/detail/vuejs-devtools/nhdogjmejiglipccpnnnanhbledajbpd?hl=en) on a Quasar 1.1.3 app but it's not clear to me how I might do so.

Normally you're supposed to do `Vue.config.devtools = true` but it's not clear to me where I'm supposed to put that code.

I tried to update quasar.conf.js per https://quasar.dev/quasar-cli-webpack/quasar-config-js to include this but it didn't help:

```
        devServer: {
            vueDevtools: true
        }
```
Any ideas?

**Comment by metalsadman** [comment]
the extension is working for me, even without that config on.
![Screen Shot 2022-04-04 at 10 03 36 AM](https://user-images.githubusercontent.com/31038838/161462218-e6cfc72d-c7a1-42a2-a380-764268232b21.png)
